apps/web/src/components/footsies.tsx

```diff
@@ -1,14 +1,19 @@
-import { Binary, Github, Twitter } from '@repo/ui/icons';
 import Link from 'next/link';
+import { Binary, Github, Twitter } from '@repo/ui/icons';
 import { ThemeButton } from './Navigation/theme-button';
+import { getScopedI18n } from '~/locales/server';
+
+export async function Footsies() {
+  const t = await getScopedI18n('footsies');
 
-export function Footsies() {
   return (
     <footer className="flex flex-col items-center gap-2 px-8 pb-12 text-sm font-light sm:px-16 sm:pb-20 sm:pt-6 md:px-0 md:py-12">
       <div className="container flex flex-col-reverse justify-between gap-2 md:flex-row md:items-end">
         <span>
-          Built with <Binary className="inline-block h-5 w-5 text-[#31bdc6]" /> by the Trash Devs
-          community.
+          {
+            // @ts-ignore It's work but the typing of lib is not updated
+            t('built', { icon: <Binary className="inline-block h-5 w-5 text-[#31bdc6]" /> })
```

Comment:
do we just need to update the lib?

Reply:
I haven't looked yet. I think you should go and make a contribution to next-international.

Reply:
cc @QuiiBz 

why does this error 👀

Reply:
JSON isn't recommended because you can't correctly infer the parameters (anything like `{something}` in a translation). That's a limitation of TypeScript: https://github.com/microsoft/TypeScript/issues/32063

If we really want to use JSON instead of TypeScript for translations, we'll have to manually create a `Locale` object to pass the correct parameters: https://next-international.vercel.app/docs/writing-locales#json

Reply:
why did we switch to json @AugustinMauroy 

i did notice the main example of next did json as well so maybe that’s why?

Reply:
oh missed that. lets revert for now or i can later

Reply:
OK !